### ISSUE TITLE
Class SMTP Not Found

### ISSUE THREAD

**Issue by wildangunawan** [ProAviaAZ/phpvms_5.5.2.72#51]
Hi. I'm currently using SMTP since my hosting block PHP mail() function to reduce spam email. They don't want to allow us to enable the mail() function and ask to use SMTP instead.

I have enabled SMTP function at local.config.php and trying to send email from admin panel, this happens:
![image](https://user-images.githubusercontent.com/35158077/89708350-f8324a80-d9a8-11ea-9cc2-61fe42411124.png)

This is my setting for local.config.php
![image](https://user-images.githubusercontent.com/35158077/89708418-86a6cc00-d9a9-11ea-89d7-9809e1b38161.png)

This error is everywhere when sending registration email, and others.

**Comment by wildangunawan** [comment]
For anyone that facing the same issue, I made some experiment and it seems successful.
Go open [DavidJClark repo](https://github.com/DavidJClark/phpvms_5.5.x), download, and extract the file.
Go to core/lib. Copy (or cut, up to you) phpmailer folder and replace ProAviaAZ's phpmailer (at the same directory, core/lib).
Then you can edit your local.config.php at /core to put all SMTP details.

**Comment by ProAviaAZ** [comment]
I believe most users are able to use the PHP mail functions - otherwise we would see a lot more issues with it. It would be best not to change the 5.5.2.72 code at this point.

Have you tried not using the above local.config settings and just have a main email address in the admin panel? Having an address in the admin panel and the above local.config section set as the default install does seems to work fine for most users.

```
# Email Settings
Config::Set('EMAIL_FROM_NAME', '');
Config::Set('EMAIL_FROM_ADDRESS', '');

Config::Set('EMAIL_USE_SMTP', false);
# Add multiple SMTP servers by separating them with ;
Config::Set('EMAIL_SMTP_SERVERS', '');
Config::Set('EMAIL_SMTP_PORT', '25');
Config::Set('EMAIL_SMTP_USE_AUTH', false);
Config::Set('EMAIL_SMTP_SECURE', 'ssl'); # must be "ssl" for Google Apps
Config::Set('EMAIL_SMTP_USER', '');
Config::Set('EMAIL_SMTP_PASS', '');
```

**Comment by wildangunawan** [comment]
As I said earlier, unfortunately our hosting disabled PHP mail() function and refused to enable it for us. They also say that we should use SMTP if we wish to send an email.

**Comment by ProAviaAZ** [comment]
Yes - I understand your host disabled PHP mail function. But for most users that is a viable option. If they have your issue, they can use your suggested fix. 

Maybe document your fix as an alternative in the phpVMS forums thread about phpVMS 5.5.2.72 
https://forum.phpvms.net/topic/26545-phpvms-55272-phpvms-552-updated-for-php-72/

**Comment by pilotchase** [comment]
> For anyone that facing the same issue, I made some experiment and it seems successful.
> Go open [DavidJClark repo](https://github.com/DavidJClark/phpvms_5.5.x), download, and extract the file.
> Go to core/lib. Copy (or cut, up to you) phpmailer folder and replace ProAviaAZ's phpmailer (at the same directory, core/lib).
> Then you can edit your local.config.php at /core to put all SMTP details.

Thank you so much for this! It works with 5.5.2.72 and just solved my issue with Mailgun after 48 hours of pulling my hair out

**Comment by ProAviaAZ** [comment]
There have been isolated cases with hosts where you may need to downgrade from the included phpMailer version to the one available in phpVMS 5.5.2 simpilot version. For most, the updated phpMailer included in phpVMS 5.5.2.72 works fine.
